components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/cache/SAMLSLOParticipantCache.java

```diff
@@ -0,0 +1,5 @@
+package org.wso2.carbon.identity.sso.saml.cache;
+
+public class SAMLSLOParticipantCache {
```

Comment:
What is this? Needs to have the license header, class header level comment. Seems you didn't import the WSO2 license header. Please do that.